### ISSUE TITLE
Blazor WASM support on .net5?

### ISSUE THREAD

**Issue by dijarkadriu** [VahidN/iTextSharp.LGPLv2.Core#88]
# Summary of the issue
When I try to generate my pdf I get PlatformNotSupportedException because System.Security.Cryptography APIs are  not supported on Blazor WebAssembly. Any work around?

## Environment

```
iTextSharp.LGPLv2.Core version: 1.7.1
.NET Core SDK version: net5.0
IDE:  Visual Studio 2019 Professional 
```

## Output:

Exception message:
Full Stack trace:
crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: System.Security.Cryptography.Algorithms is not supported on this platform.
System.PlatformNotSupportedException: System.Security.Cryptography.Algorithms is not supported on this platform.
   at System.Security.Cryptography.MD5.Create()
   at iTextSharp.text.pdf.PdfEncryption.CreateDocumentId()
   at iTextSharp.text.pdf.PdfWriter.Close()
   at iTextSharp.text.pdf.PdfDocument.Close()
   at iTextSharp.text.Document.Close()
   at FakturaEx.Client.Services.PDFGenerator.ReportPDF() in E:\Sources\FakturaEx\FakturaEx\Client\Services\PDFGenerator.cs:line 104
   at FakturaEx.Client.Services.PDFGenerator.OpenToIframe(IJSRuntime js, Int32 pagenumber, String idiFrame) in E:\Sources\FakturaEx\FakturaEx\Client\Services\PDFGenerator.cs:line 33
   at FakturaEx.Client.Pages.OrderManagement.Invoices.ClickOpenToIframe(Int32 pagetype, String idiFrame) in E:\Sources\FakturaEx\FakturaEx\Client\Pages\OrderManagement\Invoices.razor.cs:line 20
   at FakturaEx.Client.Pages.OrderManagement.Invoices.<BuildRenderTree>b__89_6(MouseEventArgs p) in E:\Sources\FakturaEx\FakturaEx\Client\Pages\OrderManagement\Invoices.razor:line 376
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task taskToHandle)

**Comment by VahidN** [comment]
Use it on the server-side of the project. Create the .pdf file in one of the action methods of the Web-API project and then download it from your wasm project.

**Comment by VahidN** [comment]
[Here is a sample](https://www.dntips.ir/file/userfile?name=BlazorWasmShowBinaryFiles.zip) about "How to show a PDF files using a Blazor WASM app".

**Comment by dijarkadriu** [comment]
I had already implemented it like this but was wondering is there any work around to generate the file in client side. But donwloading it from server is working good as well. 
Thank you for your time!

**Comment by VahidN** [comment]
Size of the `iTextSharp.LGPLv2.Core.dll` + `BouncyCastle.Crypto.dll` is 11.2 MB. It's not a wise decision to send these files to each client.

**Comment by michaelcheers** [comment]
It is fixed in the latest version: #112.
